src/LookingGlassConfig.ts

```diff
@@ -413,46 +491,77 @@ export class LookingGlassConfig extends EventTarget {
 	}
 
 	public get framebufferWidth() {
-		if (this._calibration.screenW.value < 7000) return this._viewControls.quiltResolution
-		else return 7680
+		return this.quiltResolution
+		
 	}
 
 	// number of columns
 	public get quiltWidth() {
-		if (this.calibration.screenW.value == 1536) {
-			return 8
-		}
-		else if (this.calibration.screenW.value == 3840) {
-			return 5
-
-		}
-		else if (this.calibration.screenW.value > 7000) {
-			return 5
-		}
-		else {
-			return 8
-		}
-	}
-
+		//use serial number to determine quilt width
```

Comment:
This comment is a bit misleading - this is using the serial number to determine the tile columns. I would delete it, given this is made clear in the comment above the function declaration.